flang/lib/Lower/OpenMP/ClauseProcessor.cpp

```diff
@@ -943,7 +947,7 @@ bool ClauseProcessor::processMap(
               asFortran.str(), bounds, {},
               static_cast<
                   std::underlying_type_t<llvm::omp::OpenMPOffloadMappingFlags>>(
```

Comment:
Could you perhaps add a "using" alias for the underlying type and use it here? This would improve the readability a bit.